src/engraving/dom/trill.cpp

```diff
@@ -246,6 +246,68 @@ void Trill::computeStartElement()
     }
 }
 
+PointF Trill::linePos(Grip grip, System** system) const
+{
+    bool start = grip == Grip::START;
+    bool mmRest = style().styleB(Sid::createMultiMeasureRests);
+    double graceOffset = 0.0;
+    double clefOffset = 0.0;
+
+    Segment* segment = start ? startSegment() : endSegment();
+    if (!segment) {
+        return PointF();
+    }
+    if (!start) {
```

Comment:
I'd rather write this the other way around, i.e. filter out the uninteresting case first:

```
if (start) {
  // do stuff
  // return something;
}

// Do everything else
```

Just a matter of style obvs, but avoiding long nested ifs generally helps readability :)